typegraph/node/sdk/src/tg_manage.ts

```diff
@@ -26,6 +26,9 @@ export class Manager {
       case "deploy":
         await this.#deploy();
         break;
+      case "list":
+        await this.#list();
```

Comment:
This won't be called as you use the `SerializeActionGenerator`...

Reply:
ah, ok thanks